Tella/Data/Database/Common/DatabaseConstants.swift

```diff
@@ -8,28 +8,33 @@ import SQLite3
 
 struct D {
     /* DATABASE */
+    // MARK: - DATABASE
     static let databaseName = "tella_vault.db"
     
     /* DATABASE VERSION */
 
     static let databaseVersion = 2
 
     /* DEFAULT TYPES FOR DATABASE */
+    // MARK: - DEFAULT TYPES FOR DATABASE
     static let integer = " INTEGER "
     static let text = " TEXT ";
     static let blob = " BLOB ";
     static let float = " FLOAT ";
 
     
     /* DATABASE TABLES */
-    
+    // MARK: - DATABASE TABLES
     static let tServer = "t_server"
     static let tReport = "t_report"
+    static let tUwaziServerLanguage = "t_uwazi_server_language"
```

Comment:
I think for new Uwazi server table implementation we don't need `tUwaziServerLanguage`